enterprise/templates/enterprise/base.html

```diff
@@ -36,5 +41,7 @@
     {% endblock %}
 
     {% block contents %}{% endblock %}
+
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js"></script>
```

Comment:
Note to self: I just found that we have Bootstrap v4.0.0-beta importable from the `edx-platform` given the static-based folder path `common/js/vendor/bootstrap.js`.